<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>musicWidget</class>
 <widget class="QWidget" name="musicWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>207</width>
    <height>64</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QSlider" name="horizontalSlider_music">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="musicTime">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="musicTimeMax">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>40</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="m_buttonPrevious">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>31</width>
     <height>29</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>31</width>
     <height>31</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="m_buttonPlayPause">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>0</y>
     <width>31</width>
     <height>29</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>31</width>
     <height>31</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="m_buttonNext">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>0</y>
     <width>31</width>
     <height>29</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>31</width>
     <height>31</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
